### ISSUE TITLE
Automate updates to data and contributor based on git info

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/extension-workshop#657]
Today, updating who modified a page is a manual task. This data is all in git, so we should be able to consume this via eleventy and incorporate it into the build. 

This would then remove that step from needing to be done as part of the content update.

**Comment by wagnerand** [comment]
As far as I know, this _feature_ was ported from MDN. Should we keep using it? We could just drop contributor data altogether, it's still in this github repository, where contributors are also congratulated after their patch gets merged.

**Comment by muffinresearch** [comment]
> As far as I know, this _feature_ was ported from MDN. Should we keep using it? We could just drop contributor data altogether, it's still in this github repository, where contributors are also congratulated after their patch gets merged.

Yep certainly worth revisiting the feature with Product before making changes.

**Comment by muffinresearch** [comment]
Further info for this:

All of these are 404 on the site, likely due to users change their usernames or disappearing:

- https://github.com/mdnwebdocs-bot
- https://github.com/hatem45
- https://github.com/david_ross
- https://github.com/potterwrit
- https://github.com/andymckay-github
- https://github.com/V@no
- https://github.com/hasangol
- https://github.com/Cikgu77
- https://github.com/01787500664
- https://github.com/tedmcox
- https://github.com/iwanrezpect
- https://github.com/gray_-_wolf
- https://github.com/Leif-AMO
- https://github.com/johnadungan
- https://github.com/PikadudeNo1
- https://github.com/niharikak101
- https://github.com/Zearin_Galaurum

Across the site these 17 links appear 55 times.

**Comment by caitmuenster** [comment]
+1 on automating contributor info. 

I'd prefer to keep the existing feature and automate it so external viewers have a sense of who has edited the page.

**Comment by muffinresearch** [comment]
> I'd prefer to keep the existing feature and automate it so external viewers have a sense of who has edited the page.

Am I right in thinking that "last updated by" is more important than a list of all contributors to that page?

**Comment by caitmuenster** [comment]
I agree - "last updated" is probably more critical than the full contributor list.
